docs/specs/rfc7636.rst

```diff
@@ -80,7 +80,7 @@ consider that we already have a ``session``::
     >>> from authlib.oauth2.rfc7636 import create_s256_code_challenge
     >>> code_verifier = 'generate-a-urlsafe-string'
```

Comment:
This is an invalid code verifier according to RFC7636 btw. - we shouldn't provide invalid examples IMHO. Opened #165 to track that issue.